package.json

```diff
@@ -11,7 +11,7 @@
     "clean": "rm -r dist",
     "lint": "eslint lib/ playwright/ tests/ rules/autoconsent/*.json",
     "bundle": "./build.sh",
-    "watch": "chokidar \"lib\" \"addon\" \"rules\" -c \"./build.sh\"",
+    "watch": "chokidar \"lib\" \"addon\" \"rules/autoconsent\" -c \"npm run prepublish\"",
```

Comment:
What's this change for?

Reply:
`prepublish` is needed to rebuild the JSON rules when they change. The watch path change is needed because the bundled rules end up under the same rules/ folder, so it was causing an infinite rebuild loop